test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class HomeControllerTest < ActionController::TestCase
+  test "should get home" do
```

Comment:
It seems my test is only checking if the request returns something at all. 
Would the way to check for an internal error on my side be to include some sort of "asser_response_not :error" ? I am unable to find such a method, however.